tests/manage/mcg/test_nb_mgmt_endpoint.py

```diff
@@ -0,0 +1,63 @@
+import json
+import requests
+import logging
+
+from ocs_ci.framework.testlib import tier1
+
+from ocs_ci.framework.testlib import MCGTest
+from ocs_ci.framework.testlib import skipif_ocs_version
+
+from ocs_ci.ocs.bucket_utils import retrieve_verification_mode
+
+logger = logging.getLogger(name=__file__)
+
+
+@tier1
+@skipif_ocs_version(">4.13")
```

Comment:
why dont we raise this as seperate PR against release branches <= 4.12 ?

Reply:
Thanks for brining my attention to this - this PR was created when 4.14 and 4.13 used the same OCS-CI branch so the test's purpose was to keep covering the RPC route in 4.13. However, now that they're separated we no longer need this test in 4.14, just 4.13 (in 4.12 and under we still use HTTPS calls to the RPC route instead of the MCG-CLI, so don't need that test there).

I'll start by removing this test from this PR.